src/App.css

```
@@ -85,6 +95,25 @@ td {
   background-color: #393e47;
 }
 
+.row:hover .playerPosition {
+  background-color: #30353e;
+}
+
+.playerPosition {
+  pointer-events: none;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  font-size: calc(2.2vmin);
+  color: lightgray;
+  line-height: 0;
+  width: calc(4.3vmin);
+  height: calc(4.3vmin);
+  margin: 0 auto;
+  border-radius: 100%;
+  background-color: #24272e;
+}
```

```suggestion
.playerPosition {
  pointer-events: none;
  display: flex;
  align-items: center;
  justify-content: center;
  font-size: calc(2px + 2vmin);
  color: lightgray;
  line-height: 0;
  width: calc(6px + 3.7vmin);
  height: calc(6px + 3.7vmin);
  margin: 0 auto;
  border-radius: 100%;
  background-color: #24272e;
}
```

this works a bit better with scaling/mobile